mail/src/main/java/com/sun/mail/imap/protocol/IMAPProtocol.java

```diff
@@ -761,7 +761,6 @@ public synchronized void authntlm(String authzid, String u, String p)
 	Response r = null;
 	boolean done = false;
 
-	String type1Msg = null;
```

Comment:
This became used at https://github.com/javaee/javamail/commit/081f632e1fd8ae5fa6f96a0360dc9a681c3eaa79